### PR TITLE
Add option for added symbol on Hourglass Report

### PR DESCRIPTION
**Pull request by khrys63**
A new option for Hourglass report

### CODE REVIEW

**Comment by khrys63** [comment]
image
for this result :
image

**Comment by Nick-Hall** [comment]
This enhancement needs to use the [`Symbols`](https://github.com/gramps-project/gramps/blob/master/gramps/gen/utils/symbols.py#L35) class.

File: gramps/plugins/graph/gvhourglass.py [hunk 1]
```diff
@@ -444,3 +455,8 @@ def add_menu_options(self, menu):
         ahnentafelnumvisible.set_help(
             _("Show Sosa / Sosa-Stradonitz / Ahnentafel number under all others informations."))
         menu.add_option(category_name, "ahnentafelnum", ahnentafelnumvisible)
+
+        datesymbol = BooleanOption(_("Add Symbol for date"), False) # 10826
+        datesymbol.set_help(
+            _("Add a symbol for date. o for birth, + for decease and x for wedding."))
```
Comment (wroldwiedbwe) [inline gramps/plugins/graph/gvhourglass.py:461]: Suggested corrections: 

* Add a symbol for date.  >  Show symbol for events.

* decease > decease**d**

* wedding  > marriage

**Comment by wroldwiedbwe** [comment]
You might also want to use the the symbols from the built in "Genealogical Symbols" option that the user selects instead of the ones you have hard-coded?

> https://gramps-project.org/wiki/index.php/Gramps_5.1_Wiki_Manual_-_Settings#Genealogical_Symbols

File: gramps/plugins/graph/gvhourglass.py (hunk 1 above)
Comment (khrys63) [reply gramps/plugins/graph/gvhourglass.py:461]: nice idea, 
English is not my best language :)

**Comment by khrys63** [comment]
> You might also want to use the the symbols from the built in "Genealogical Symbols" option that the user selects instead of the ones you have hard-coded?
> 
> > https://gramps-project.org/wiki/index.php/Gramps_5.1_Wiki_Manual_-_Settings#Genealogical_Symbols

I use this list : https://geneafinder.com/blog?id=13:113
birth : o or *
decease : +
marriage : X or oo

what is the real standard?

**Comment by wroldwiedbwe** [comment]
> what is the real standard?

No real standard, have a read of  https://www.tamurajones.net/GenealogySymbols.xhtml

File: gramps/plugins/graph/gvhourglass.py [hunk 2]
```diff
@@ -456,7 +456,7 @@ def add_menu_options(self, menu):
             _("Show Sosa / Sosa-Stradonitz / Ahnentafel number under all others informations."))
         menu.add_option(category_name, "ahnentafelnum", ahnentafelnumvisible)
 
-        datesymbol = BooleanOption(_("Add Symbol for date"), False) # 10826
+        datesymbol = BooleanOption(_("Show symbol for events"), False) # 10826
         datesymbol.set_help(
-            _("Add a symbol for date. o for birth, + for decease and x for wedding."))
+            _("Add a symbol for date. o for birth, + for deceased and x for marriage."))
```
Comment (wroldwiedbwe) [inline gramps/plugins/graph/gvhourglass.py:461]: Missed one:   

*  Add a symbol for date. > Show symbol for events.

Comment (khrys63) [reply gramps/plugins/graph/gvhourglass.py:466]: sure?

**Comment by khrys63** [comment]
image

File: gramps/plugins/graph/gvhourglass.py (hunk 2 above)
Comment (PQYPLZXHGF) [reply gramps/plugins/graph/gvhourglass.py:466]: Yes @wroldwiedbwe English phrasing is correct ; as you are not adding a date on showing a symbol beside the three event types.  

@khrys63 Haven't tested, but hopefully the symbols don't show if the events don't exist or are blank?

Comment (khrys63) [reply gramps/plugins/graph/gvhourglass.py:466]: with last commit it's good :)

image

**Comment by SNoiraud** [comment]
Is it really useful to have an option ?
In the preferences symbols tab, we have the possibility to use or not the symbols.
If we don't use the symbols, we can define the replacement characters.

**Comment by Nick-Hall** [comment]
@SNoiraud The option is useful because there are two formats:  (1 jan 1900 - 2 Feb 1950) and (* 1 jan 1900 + 2 Feb 1950).

I'm getting some problems with the example database though:

Garner, Joseph (* 1792 ✝ )
Daniels, Phoebe (* )

**Comment by Nick-Hall** [comment]
Rebased and made some minor changes to the text.

**Pull request merged**